testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml

```diff
@@ -1,6 +1,6 @@
 databases:
   hive:
     host: hadoop-master
-  presto:
+  trino:
```

Comment:
This is an override, you also have to modify the base config: https://github.com/trinodb/trino/blob/master/testing/trino-product-tests/src/main/resources/tempto-configuration.yaml and the default database used for executing queries in https://github.com/trinodb/trino/blob/master/testing/trino-product-tests/src/main/java/io/trino/tests/product/utils/QueryExecutors.java#L35

Reply:
also https://github.com/trinodb/trino/blob/master/testing/trino-product-tests/src/main/resources/tempto-configuration.yaml#L33 would likely need to be changed.